src/boot/efi/log.c

```diff
@@ -5,6 +5,7 @@
 #include "proto/simple-text-io.h"
 #include "util.h"
 
+LogLevel max_log_level = LOG_WARNING;
```

Comment:
so in userspace we show everything starting with LOG_INFO. I would fine it utterly confusing to depart from that in sd-boot. That log_info() would be hidden in sd-boot, but visible in userspace by default is very very confusing.